ephios/plugins/baseshiftstructures/templates/baseshiftstructures/participation_card_inline.html

```diff
@@ -2,9 +2,14 @@
 {% load user_extras %}
 {% load i18n %}
 
+
 <div id="participant-popover-title-{{ participation.id }}" class="d-none">
     <span class="text-dark pe-1">{{ participation.participant.icon }}</span>
-    {{ participation.participant }}
+    {% if  participation.show_participant_data %}
+        {{ participation.participant }}
+    {% else %}
+        {% translate "Anonymous" %}
```

Comment:
I think we should use a more friendly term, IMO its weird to see that you will share the shift with "Anonymous". One idea would be to disclose the relevant qualifications (reads as: I will share the KTW with another RS). If we don't want to expose this as well we should at least use a more friendly term